### ISSUE TITLE
No pointer when permission to show pointer is deactivated

### ISSUE THREAD

**Issue by Syrious** [xaukael/pointers#1]
As a GM I don't want my players to see my default pointer all the time (in case I have to move invisible creatures around) so I deactivated the showing of the GM pointer (see screenshot). But this has the side-effect of disabling the pointer from this module, too.

Can this be fixed?

<img width="501" alt="image" src="https://github.com/xaukael/pointers/assets/8050391/222923fd-6518-4bd3-a754-48c8bcf6c62d">

**Comment by xaukael** [comment]
I just tested this and it does not seem to be the case for v10 or v11. What Foundry version and module version are you on?

**Comment by Syrious** [comment]
Thank you for the answer. I'm using v11.

I have just checked it again. It's probably not a permission problem as I thought before. When I created a new world and only activated your module, everything worked as expected. So I experimented a bit and found out that it doesn't work in conjunction with [Monk's Common Display](https://foundryvtt.com/packages/monks-common-display).

On the GM side the pointer is visible but not to connected clients.
I previously used [PnP](https://foundryvtt.com/packages/pointer/) that worked together with Monk's Common Display.

Thanks for looking into it!
